stubs/livewire/resources/views/api/api-token-manager.blade.php

```diff
@@ -13,7 +13,7 @@
             <!-- Token Name -->
             <div class="col-span-6 sm:col-span-4">
                 <x-jet-label for="name" value="Token Name" />
-                <x-jet-input id="name" type="text" class="mt-1 block w-full" wire:model="createApiTokenForm.name" autofocus />
+                <x-jet-input id="name" type="text" class="mt-1 block w-full" wire:model.defer="createApiTokenForm.name" autofocus />
```

Comment:
I've deferred all text inputs. I suppose you could defer ALL inputs, but this will save on the majority of requests. Up to you.